assets/js/admin.js

```diff
@@ -48,6 +48,9 @@
 			tolerance: 'pointer',
 			forcePlaceholderSize: true,
 			cursorAt: { top: 40 },
+			start: function( e, ui ){
+				ui.placeholder.height( ui.helper.height() );
```

Comment:
@EvanHerman I noticed there is a tiny jump that happens when you start dragging. I think it's because `height()` does not include padding and borders, but `outerHeight()` does:

```js
ui.placeholder.outerHeight( ui.helper.height() );
```